CIP-0120/README.md

```diff
@@ -0,0 +1,411 @@
+off-chain storage best practices or use of rich text styling.
+
+By formalizing the form of the constitution and its iterations, 
+we aim to promote its longevity and accessibility.
+This is essential to ensure the effectiveness of the CIP-1694 governance model.
+
+This standard will impact how Ada holders read the Constitution but 
+the main stakeholders for this are the tool makers who wish to read, 
+render and write a constitution.
+
+### Safety
+
+Without describing best practices for the form and handling of the constitution,
+ we risk the constitution document being stored in an insecure manner.
+By storing the constitution on a decentralized platform, 
+we can ensure its immutability and permissionless access.
+This is a step to improve the longevity and accessibility of each constitution 
+iteration.
+
+### Interoperability
+
+By defining a file extension and formatting rules for the constitution we 
+ensure that tooling reading and writing the constitution will be interoperable.
+Furthermore we aim to make the role of constitution iteration comparison tools 
+easier, by minimizing formatting and style changes between iterations.
+This will reduce compatibility issues between tools, 
+promoting the accessibility of the constitution.
+
+### Usability
+
+Rich text formatting greatly enhances the readability of text, 
+especially in large complex documents.
+Without the ability to format text, it could easily become cumbersome to read, 
+negatively effecting the accessability of the Cardano constitution.
+
+## Specification
+
+The following specification SHOULD be applied to all constitution iterations.
+This standard could be augmented in the future via a separate CIP which aims to 
+replace this one.
+
+### Terminology
+
+#### Capitalisation of key terms
+
+To avoid unnecessary edits, and therefore checksum changes, 
+constitution authors MUST follow the following standard English capitalisation 
+rules unless a translation language indicates otherwise:
+
+`Constitution` vs. `constitution`
+
+- The Constitution in effect — either the "initial" one or any updated version — is unique and therefore capitalised ("Constitution") as a proper noun.
+- Draft or proposed constitutions are not unique & therefore are not capitalised ("constitution") as a common noun.
+- "Cardano Constitution" is a very specific proper noun phrase (also a title) and so each word is capitalised.
+- The phrase "the Constitution", unless used non-specifically (e.g. "the constitution that voters prefer"), would generally be assumed to be _the_ Constitution and capitalised as a proper noun.
+
+`Ada` vs. `ada` vs. `ADA`
+
+- `Ada` is the currency, while `ada` indicates units of that currency (e.g. "Ada holders can accumulate more ada to increase their influence.")
+- `ADA` is the trading symbol (e.g. "Fluctuations in ADA might influence decisions about the Treasury.")
+
+### Characters
+
+The constitution text MUST only contain UTF-8 characters.
```

Comment:
Can this be more specific.  I've looked up UTF-8, and it appears to be a way to encode character numbers into binary.  This should be a character set.  ASCII, for example, shows a discrete set of characters that a writer can use when drafting.

Reply:
@Ryun1 - @daniellestanko raises a vital point since UTF-8 will include emojis, "dingbats", and the like because of extending up to 4 bytes with a very broad range of characters; see https://en.wikipedia.org/wiki/UTF-8#Encoding

I'd be tempted to restrict to the first 1920 code points (i.e. the 2-byte characters of UTF-8) but this would be biased toward Western languages and would certainly prevent e.g. a constitution translation in Chinese... which would require the 3-byte characters of the [Basic Multilingual Plane](https://en.wikipedia.org/wiki/Plane_(Unicode)#Basic_Multilingual_Plane) which, counter to your original goal, also includes graphical symbols.

Therefore I think you will have to include a subjective specification (unfortunately, perhaps unenforceable by text filters or anything but a manual reading) like:
```suggestion
The constitution text MUST only contain printable text characters in UTF-8 encoding.
```
The Multilingual Plane also includes mathematical symbols and therefore would accommodate constitutions which, for whatever reason, have mathematical expressions in their text (I think we are bound to see this at some point).